### PR TITLE
Unpin protobuf 3.20.1

### PR DESCRIPTION
**Pull request by fxmarty**
Fixes https://github.com/huggingface/optimum/issues/733

Feel free to pin in subpackages if necessary.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -34,14 +34,14 @@
         "onnxruntime>=1.9.0",
         "datasets>=1.2.1",
         "evaluate",
-        "protobuf==3.20.1",
+        "protobuf>=3.20.1",
```
Comment (regisss) [inline setup.py:37]: Shouldn't we put `>=3.20.2` since v3.20.1 is the version with the security issue?

Comment (fxmarty) [reply setup.py:37]: Not sure, maybe other package require 3.20.1.

Comment (regisss) [reply setup.py:37]: Fair enough

**Review by regisss — APPROVED** [review]
LGTM!

**Pull request merged**